CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ As a growing community project, we always appreciate contributions. We're lookin
 2. **Code:** enhance the site with HTML/CSS/JS elements, create CI/CD pipelines, perform general maintenance
 3. **Design:** create new infographics, assets, improve existing formatting
 
-Whether this is your first time contributing or you're an OSS pro, we would love to have your help improving <a href="https://ephemeralenvironments.io" target="_blank">EphemeralEnvironments.io</a>!
+Whether this is your first time contributing or you're an OSS pro, we would love to have your help improving [EphemeralEnvironments.io](https://ephemeralenvironments.io)!
```

Comment:
This is formatted using the html element so the link opens in a new tab

Reply:
ok just read the docs, no native way to use markdown to open a new tab, must use inline html